pkg/gatewayconfigurationserver/v2/server.go

```diff
@@ -75,13 +78,18 @@ func WithTheThingsGatewayConfig(config TheThingsGatewayConfig) Option {
 	}
 }
 
-const compatAPIPrefix = "/api/v2"
-
 // RegisterRoutes implements the web.Registerer interface.
-func (s *Server) RegisterRoutes(srv *web.Server) {
-	group := srv.Group(compatAPIPrefix, s.normalizeAuthorization)
-	group.GET("/gateways/:gateway_id", s.handleGetGateway)
-	group.GET("/frequency-plans/:frequency_plan_id", s.handleGetFrequencyPlan)
+func (s *Server) RegisterRoutes(server *web.Server) {
+	router := server.Prefix("/api/v2/").Subrouter()
```

Comment:
Good to keep in mind that this claims all of `/api/v2/` to this SubRouter (at least, I think it does)

Reply:
Yeah, good one, will use routes only. This can only backfire in the future.